src/main/java/Interpreter.java

```diff
@@ -0,0 +1,23 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Interpreter {
+
+
+    public List<String> getOperands(String inputExpression) {
+        return checkFirstValue(inputExpression);
+    }
+
+    public List<String> getOperators(String inputExpression) {
+        return new ArrayList<>(List.of(inputExpression.split("[0-9]+")));
+    }
+
+    private List<String> checkFirstValue(String inputExpression) {
+        List<String> operands = new ArrayList<>(List.of(inputExpression.split("[+-]")));
+        if (getOperators(inputExpression).get(0).equals("-")){
```

Comment:
This could be done once, before calling this function instead of forcing a recursive call. Recursive calls should be avoided if possible as they lower code readability among other tihngs.

Reply:
as in previous comment i've delete this function